common/src/main/java/earth/terrarium/adastra/client/screens/base/MachineScreen.java

```diff
@@ -129,95 +131,28 @@ public void render(@NotNull GuiGraphics graphics, int mouseX, int mouseY, float
-                ((PressableImageButton) button).setTexture(next.icon());
-            },
-            getRedstoneControlTooltip(entity.getRedstoneControl())
-        ));
-    }
-
-    public void addSideConfigButton(int xOffset, int yOffset) {
-        addRenderableWidget(new PressableImageButton(this.leftPos + xOffset, this.topPos + yOffset, 18, 18, 0, 0, 18, GuiUtils.SETTINGS_BUTTON, 18, 54,
-            button -> toggleSideConfig(),
-            ConstantComponents.SIDE_CONFIG
-        ));
-    }
-
-    protected void toggleSideConfig() {
-        this.menu.togglePlayerInvSlots();
-        showSideConfig = !showSideConfig;
-        sideConfigButtons.forEach(all -> all.forEach(b -> b.visible = false));
-        sideConfigButtons.get(arrowIndex).forEach(b -> b.visible = showSideConfig);
-        resetToDefaultButton.visible = showSideConfig;
-        previousArrow.visible = showSideConfig;
-        nextArrow.visible = showSideConfig;
-    }
 
-    public void drawEnergyBar(GuiGraphics graphics, int mouseX, int mouseY, int xOffset, int yOffset, EnergyContainer energy, long energyDifference) {
-        GuiUtils.drawEnergyBar(
-            graphics, mouseX,
-            mouseY, font,
-            this.leftPos + xOffset,
-            this.topPos + yOffset,
-            energy,
-            TooltipUtils.getEnergyDifferenceComponent(energyDifference),
-            TooltipUtils.getMaxEnergyInComponent(energy.maxInsert()),
-            TooltipUtils.getMaxEnergyOutComponent(energy.maxExtract()));
-    }
-
-    public void drawFluidBar(GuiGraphics graphics, int mouseX, int mouseY, int xOffset, int yOffset, FluidContainer fluid, int tank, long fluidDifference) {
-        int x = this.leftPos + xOffset;
-        int y = this.topPos + yOffset;
-        GuiUtils.drawFluidBar(
-            graphics, mouseX,
-            mouseY, font,
-            x, y,
-            fluid, tank,
-            TooltipUtils.getFluidDifferenceComponent(fluidDifference),
-            !fluid.isEmpty() ? ConstantComponents.CLEAR_FLUID_TANK : Component.empty());
-
-        clearOnClicks.add(new ClearOnClick(x + 6, y - 31, x + 22, y + 17, tank));
-    }
-
-    public void drawOptionsBar(GuiGraphics graphics, int xOffset, int yOffset) {
-        graphics.blit(GuiUtils.OPTIONS_BAR, this.leftPos + xOffset, this.topPos + yOffset, 0, 0, 72, 30, 72, 30);
+        if (texture != null) {
+            graphics.blit(texture, slot.x - 1, slot.y - 1, 0, 0, 18, 18, 18, 18);
+        }
     }
 
-    public void drawSmallOptionsBar(GuiGraphics graphics, int xOffset, int yOffset) {
-        graphics.blit(GuiUtils.SMALL_OPTIONS_BAR, this.leftPos + xOffset, this.topPos + yOffset, 0, 0, 30, 30, 30, 30);
+    @Override
+    protected void renderLabels(GuiGraphics graphics, int mouseX, int mouseY) {
+        graphics.drawString(font, this.title, this.titleLabelX, this.titleLabelY, 0x2a262b, false);
+        Component label = canConfigure() ? this.sideConfigWidget.getMessage() : this.playerInventoryTitle;
```

Comment:
Side config title covers gui elements in solar panel, nasa workbench, cryo freezer, and especially the water pump.
<img width="290" alt="Screenshot 2023-12-31 at 1 12 26 AM" src="https://github.com/terrarium-earth/Ad-Astra/assets/61668546/9de16b07-0117-4ece-b866-9051b5d627c4">

Reply:
All these guis never fit an inventory title that wasnt English so we need to extend them to account for that